skimage/_shared/tests/test_testing.py

```diff
@@ -109,6 +109,7 @@ def change_state3():
     assert len(state) == 6
 
 
+@pytest.mark.skipif(is_wasm, reason="Cannot run parallel code in WASM")
```

Comment:
I'm guessing this was because the earlier Node.js-based test runner did not directly inherit the configuration for `pytest` from `pyproject.toml`, because the wheel and the test dependencies were mounted in a custom directory that was isolated from the system's file system, which meant that `pytest` went ahead with its defaults when running the tests.

In the [Pyodide job for another PR](https://github.com/scikit-image/scikit-image/actions/runs/10654875233/job/29531715274?pr=7523#step:8:369), I notice that this is currently emitted as a warning, but it now gets converted to an error with the new virtual environment, due to the existence of the `filterwarnings = ["error"]` key in the `[tool.pytest.ini_options]` table. It has to be either filtered/ignored there, or skipped. IMO, it's better to skip it conditionally, since this is specific to Pyodide, rather than to filter the warning for other platforms (unless there is a way in `pytest` to let it conditionally filter the warnings, too).

Reply:
Okay, that explanation makes sense. Thanks for delving into it. :+1: